### PR TITLE
[DEVEXP-457] Create application from source in the management console

### PR DESCRIPTION
**Pull request by jcantrill**
cc @bparees

### CODE REVIEW

File: dev_guide/new_app.adoc [hunk 1]
```diff
@@ -70,3 +70,42 @@ values defined by the template:
 $ osc new-app ruby-helloworld-sample -p ADMIN_USERNAME=myuser,ADMIN_PASSWORD=mypass
 ----
 
+== Using the Management Console
```
Comment (adellape) [inline dev_guide/new_app.adoc]: Given this new heading, the above "Usage Examples" could probably become "Using the CLI".

**Comment by adellape** [comment]
:+1:

**Comment by jcantrill** [comment]
@adellape Fixed header per your comment

**Comment by adellape** [comment]
Thanks, will merge after build passes.

**Comment by adellape** [comment]
@jcantrill Merging.

**Pull request merged**